.github/workflows/nightly.yml

```diff
@@ -55,9 +55,6 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - uses: eWaterCycle/setup-singularity@v7        # for containerized code test
-      with:
-        singularity-version: 3.8.7
 
     - name: Install system dependencies
       run: sudo apt update && sudo apt install postgresql
```

Comment:
I wonder if we should be installing graphviz here?

Reply:
Why? it seems the nightly pytests don't need graphviz.